src/app/rpmostree-builtin-status.c

```diff
@@ -79,6 +92,17 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
   const char *bold_suffix = is_tty ? "\x1b[0m" : "";
   const char *red_prefix = is_tty ? "\x1b[31m" : "";
   const char *red_suffix = is_tty ? "\x1b[22m" : "";
+  GVariant* txn = get_active_txn (sysroot_proxy);
+
+  if (txn)
+    {
+      const char *method, *sender, *path;
+      g_variant_get (txn, "(&s&s&s)", &method, &sender, &path);
+      g_print ("State: transaction: %s %s %s\n", method, sender, path);
+    }
+  else
+    g_print ("State: idle\n");
+  g_print ("\n");
```

Comment:
This is an interesting idea, and I like it.
A `Deployments:` header might be nice as well to make the overall output more cohesive, e.g.:

```
State: idle
Deployments:

  fedora-atomic:fedora-atomic/f23/x86_64/docker-host
       Version: 23.137 (2016-06-14 21:40:52)
        Commit: 0e686e114410cbc7e04713970c417338efc0d657779b428bff23ad571bfbc927
        OSName: fedora-atomic
  GPGSignature: (unsigned)

* fedora-atomic:fedora-atomic/f23/x86_64/docker-host
       Version: 23.131 (2016-06-07 01:00:21)
        Commit: aadcfe51acd85e8cf2554b6a3c952d8335a934d219bbb537fec10ce0fde7dc06
        OSName: fedora-atomic
  GPGSignature: (unsigned)
      Unlocked: development
```

WDYT?